src/datepicker-input/datepicker-input.component.ts

```diff
@@ -40,9 +40,7 @@ import {
 					[disabled]="disabled"
 					[attr.data-invalid]="(invalid ? true : null)"
 					(change) = "valueChange.emit(dateInput.value)"/>
-					<div *ngIf="invalid" class="bx--form-requirement">
-						{{invalidText}}
-					</div>
+					<div *ngIf="invalid" class="bx--form-requirement" [innerHtml]="invalidText"></div>
```

Comment:
I'm not sure about this change. Now we're encouraging passing around raw HTML strings, which is really un-ergonomic and doesn't allow for any Angular template features ... It would be better if it was plain text or a template (for custom situations)